CONTRIBUTING.md

```diff
@@ -37,27 +37,29 @@ to avoid duplication of effort.
 The process of submitting a Pull Request is relatively simple, following a 
 [rebase workflow](http://kensheedlo.com/essays/why-you-should-use-a-rebase-workflow/).  Please use the
 following pattern:
+
 1. Create a new branch for your fix/feature
 1. Make your changes
-1. Rebase onto the upstream
-1. Run the test suite
-1. Squash your commits
+1. Run the test suite *(if failed, goto 2)*
+1. Squash your commits *(re-run test suite)*
+1. Rebase onto the upstream *(re-run test suite)*
```

Comment:
I see your point, and I guess not having tested the code post-rebase would be alright, since the PR builder does just that.  In addition, the italicized blocks encourage running the suite after rebase.  :+1: